test/hotspot/jtreg/compiler/vectorization/TestRotateByteAndShortVector.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Why you need this copyright?
Maybe, we can remove it.

Reply:
Yes, just remove it.